src/ucs/stats/serialization.c

```diff
@@ -232,6 +234,10 @@ ucs_stats_serialize_binary_recurs(FILE *stream, ucs_stats_node_t *node,
     ucs_stats_clsid_t *elem, search;
     ucs_stats_node_t *child;
     uint8_t sentinel;
+    ucs_stats_filter_node_t *filter_node;
+    ucs_stats_counter_t filtered_counters[64] = {};
```

Comment:
plz, move initialized field to the very beginning of the func

Reply:
Fixed.